packages/litexa/src/parser/skill.coffee

```diff
@@ -363,11 +364,12 @@ class lib.Skill
     do =>
       output.push "litexa.sayMapping = ["
       lines = []
-      for source, target of @sayMapping
-        source = source.replace(/'/g, '\\\'')
-        target = target.replace(/'/g, '\\\'')
-        lines.push "  { test: new RegExp(' #{source}','gi'), change: ' #{target}' }"
-        lines.push "  { test: new RegExp('#{source} ','gi'), change: '#{target} ' }"
+      for mapping in @sayMapping
+        source = mapping.source.replace /'/g, '\\\''
+        target = mapping.target.replace /'/g, '\\\''
+        language = mapping.language
+        lines.push "  { test: new RegExp(' #{source}','gi'), change: ' #{target}', language: '#{language}' }"
+        lines.push "  { test: new RegExp('#{source} ','gi'), change: '#{target} ', language: '#{language}' }"
```

Comment:
Odd to have the naming suddenly flipped to `test/change` here. Would align the naming with the function parameters. For instance, if you were to go with `from/to` you could use something like `fromRegex` and `toRegex`.